packages/snap/src/index.ts

```diff
@@ -240,8 +247,8 @@ async function getOptimalFeeCurrency(tx: CeloTransactionRequest, wallet: CeloWal
   const gasLimit = (await wallet.estimateGas(tx)).mul(5)
   const celoBalance = await wallet.getBalance();
   const tokenAddresses = await feeCurrencyWhitelistContract.getWhitelist();
-
-  if (gasLimit.add(tx.value ?? 0) >= celoBalance) {
+  const GasLimitPlusValue = gasLimit.add(tx.value ?? 0);
```

Comment:
nit but its weird to have. variable be Capitalized. ususally this is only for classes in js

Reply:
yes agreed...just coding at 11pm 😴